File: frontend/notes-app/src/app/models/Note.ts
export class Note {
    id: number;
    title: string;
    description: string;
    category: string;
    archived: boolean;

    constructor() {
      this.id = 0;
      this.title = '';
      this.description = '';
      this.category = '';
      this.archived = false;
    }

  }
  

File: backend/notes.app/src/main/java/com/gestionnotas/notes/app/models/Note.java
package com.gestionnotas.notes.app.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@AllArgsConstructor
@NoArgsConstructor
public class Note {
    private Long id;
    private String title;
    private String description;
    private String category;
    private boolean isArchived;
}